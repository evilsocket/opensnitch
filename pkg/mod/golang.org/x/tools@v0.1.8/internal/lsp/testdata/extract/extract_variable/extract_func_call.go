package extract

import "strconv"

func _() {
	x0 := append([]int{}, 1) //@suggestedfix("append([]int{}, 1)", "refactor.extract")
	str := "1"
	b, err := strconv.Atoi(str) //@suggestedfix("strconv.Atoi(str)", "refactor.extract")
}
