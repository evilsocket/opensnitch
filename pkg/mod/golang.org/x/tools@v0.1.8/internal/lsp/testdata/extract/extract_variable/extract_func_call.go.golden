-- suggestedfix_extract_func_call_6_7 --
package extract

import "strconv"

func _() {
	x0 := append([]int{}, 1)
	a := x0 //@suggestedfix("append([]int{}, 1)", "refactor.extract")
	str := "1"
	b, err := strconv.Atoi(str) //@suggestedfix("strconv.Atoi(str)", "refactor.extract")
}

-- suggestedfix_extract_func_call_6_8 --
package extract

import "strconv"

func _() {
	x := append([]int{}, 1)
	x0 := x //@suggestedfix("append([]int{}, 1)", "refactor.extract")
	str := "1"
	b, err := strconv.Atoi(str) //@suggestedfix("strconv.Atoi(str)", "refactor.extract")
}

-- suggestedfix_extract_func_call_8_12 --
package extract

import "strconv"

func _() {
	x0 := append([]int{}, 1) //@suggestedfix("append([]int{}, 1)", "refactor.extract")
	str := "1"
	x, x1 := strconv.Atoi(str)
	b, err := x, x1 //@suggestedfix("strconv.Atoi(str)", "refactor.extract")
}

